<documentation title="Silenced Errors">
    <standard>
    <![CDATA[
    Suppressing Errors is not allowed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: isset() is used to verify that a variable exists before trying to use it.">
        <![CDATA[
if (<em>isset($foo)</em> && $foo) {
    echo "_PhpScoper7faa8deb0d3c\Hello\n";
}
        ]]>
        </code>
        <code title="Invalid: Errors are suppressed.">
        <![CDATA[
if (<em>@</em>$foo) {
    echo "_PhpScoper7faa8deb0d3c\Hello\n";
}
        ]]>
        </code>
    </code_comparison>
</documentation>
