<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScopereb9508917a55\Vendor\Package\SomeNamespace\{
    _PhpScopereb9508917a55\SubnamespaceOne\ClassA,
    _PhpScopereb9508917a55\SubnamespaceOne\ClassB,
    _PhpScopereb9508917a55\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScopereb9508917a55\Vendor\Package\SomeNamespace\{
    <em>_PhpScopereb9508917a55\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScopereb9508917a55\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
