<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoperdf6a0b341030\Vendor\Package\SomeNamespace\{
    _PhpScoperdf6a0b341030\SubnamespaceOne\ClassA,
    _PhpScoperdf6a0b341030\SubnamespaceOne\ClassB,
    _PhpScoperdf6a0b341030\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoperdf6a0b341030\Vendor\Package\SomeNamespace\{
    <em>_PhpScoperdf6a0b341030\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoperdf6a0b341030\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
