<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoperbcc0395698f8\Vendor\Package\SomeNamespace\{
    _PhpScoperbcc0395698f8\SubnamespaceOne\ClassA,
    _PhpScoperbcc0395698f8\SubnamespaceOne\ClassB,
    _PhpScoperbcc0395698f8\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoperbcc0395698f8\Vendor\Package\SomeNamespace\{
    <em>_PhpScoperbcc0395698f8\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoperbcc0395698f8\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
