<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoperaa402dd1b1f1\Vendor\Package\SomeNamespace\{
    _PhpScoperaa402dd1b1f1\SubnamespaceOne\ClassA,
    _PhpScoperaa402dd1b1f1\SubnamespaceOne\ClassB,
    _PhpScoperaa402dd1b1f1\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoperaa402dd1b1f1\Vendor\Package\SomeNamespace\{
    <em>_PhpScoperaa402dd1b1f1\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoperaa402dd1b1f1\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
