<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoper57c79939064b\Vendor\Package\SomeNamespace\{
    _PhpScoper57c79939064b\SubnamespaceOne\ClassA,
    _PhpScoper57c79939064b\SubnamespaceOne\ClassB,
    _PhpScoper57c79939064b\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoper57c79939064b\Vendor\Package\SomeNamespace\{
    <em>_PhpScoper57c79939064b\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoper57c79939064b\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
