<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoperc8fea59b0cb1\Vendor\Package\SomeNamespace\{
    _PhpScoperc8fea59b0cb1\SubnamespaceOne\ClassA,
    _PhpScoperc8fea59b0cb1\SubnamespaceOne\ClassB,
    _PhpScoperc8fea59b0cb1\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoperc8fea59b0cb1\Vendor\Package\SomeNamespace\{
    <em>_PhpScoperc8fea59b0cb1\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoperc8fea59b0cb1\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
