<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoper78af57a363a0\Vendor\Package\SomeNamespace\{
    _PhpScoper78af57a363a0\SubnamespaceOne\ClassA,
    _PhpScoper78af57a363a0\SubnamespaceOne\ClassB,
    _PhpScoper78af57a363a0\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoper78af57a363a0\Vendor\Package\SomeNamespace\{
    <em>_PhpScoper78af57a363a0\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoper78af57a363a0\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
