<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use _PhpScoperc5e9eb67638f\Vendor\Package\SomeNamespace\{
    _PhpScoperc5e9eb67638f\SubnamespaceOne\ClassA,
    _PhpScoperc5e9eb67638f\SubnamespaceOne\ClassB,
    _PhpScoperc5e9eb67638f\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use _PhpScoperc5e9eb67638f\Vendor\Package\SomeNamespace\{
    <em>_PhpScoperc5e9eb67638f\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    _PhpScoperc5e9eb67638f\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
